ros2interface/ros2interface/verb/package.py

```diff
@@ -21,16 +21,41 @@ class PackageVerb(VerbExtension):
     """Output a list of available interface types within one package."""
 
     def add_arguments(self, parser, cli_name):
+        parser.add_argument(
+            '-m', '--only-msgs', action='store_true',
+            help='Print out only the message types')
+
+        parser.add_argument(
+            '-s', '--only-srvs', action='store_true',
+            help='Print out only the service types')
+
+        parser.add_argument(
+            '-a', '--only-actions', action='store_true',
+            help='Print out only the action types')
+
         arg = parser.add_argument(
             'package_name',
             help="Name of the ROS package (e.g. 'example_interfaces')")
         arg.completer = package_name_completer
 
     def main(self, *, args):
+        types_to_filter = []
+        if args.only_msgs or args.only_srvs or args.only_actions:
+            if not args.only_msgs:
+                types_to_filter.append('msg')
+            if not args.only_srvs:
+                types_to_filter.append('srv')
+            if not args.only_actions:
+                types_to_filter.append('action')
+
         try:
             interfaces = get_interfaces([args.package_name])
         except LookupError as e:
             return str(e)
         for package_name in sorted(interfaces):
             for interface_name in interfaces[package_name]:
+                if types_to_filter:
+                    interface_type = interface_name[:interface_name.index('/')]
+                    if interface_type in types_to_filter:
+                        continue
```

Comment:
I don't feel like this is the right way to filter here.

In particular, while we almost certainly aren't going to change the prefix for `msg`, `srv`, and `action`, we also have dedicated functions for getting this information.  Thus, I think we should refactor this whole block to be something like:

```
interfaces = []
if not args.only_msgs and not args.only_srvs and not args.only_actions:
    interfaces = get_interfaces([args.package_name])
else:
    if args.only_msgs:
        interfaces += get_message_interfaces([args.package_name])
    if args.only_srvs:
        interfaces += get_service_interfaces([args.package_name])
    if args.only_actions:
        interfaces += get_action_interfaces([args.package_name])

for package_name in sorted(interfaces):
    for interface_name in interfaces[package_name]:
        print(f'{package_name}/{interface_name}')

```